views/includes/footer.hbs

```diff
@@ -28,3 +28,4 @@
   </a>
   </ul>
 <footer>
+<script async src="https://www.google-analytics.com/analytics.js"></script>
```

Comment:
Would it make sense to create this programmatically in analytics.js to have it fully encapsulated?